packages/terra-badge/docs/themeable-variables.md

```diff
@@ -42,7 +42,7 @@ Map Structure
 
 | Name              | Default value             | Usage                                        |
 |-------------------|---------------------------|----------------------------------------------|
-| background-color  | $terra-primary-color-100  | Set the label primary state background color |
+| background-color  | $terra-blue-100           | Set the label primary state background color |
```

Comment:
I think primary still makes sense here. We'll still have a primary badge /  button. With this change, instead of using a scheme alias variable for context color, we are mapping directly to the source color.